echallan-services/src/main/java/org/egov/echallan/model/Address.java

```diff
@@ -0,0 +1,330 @@
+    this.region = region;
+    return this;
+  }
+
+  public String getRegion() {
+    return region;
+  }
+  public void setRegion(String region) {
+    this.region = region;
+  }
+  public Address state(String state) {
+    this.state = state;
+    return this;
+  }
+
+  public String getState() {
+    return state;
+  }
+  public void setState(String state) {
+    this.state = state;
+  }
+  public Address country(String country) {
+    this.country = country;
+    return this;
+  }
+
+  public String getCountry() {
+    return country;
+  }
+  public void setCountry(String country) {
+    this.country = country;
+  }
+  public Address pincode(String pincode) {
+    this.pincode = pincode;
+    return this;
+  }
+
+  public String getPincode() {
+    return pincode;
+  }
+  public void setPincode(String pincode) {
+    this.pincode = pincode;
+  }
+  public Address additionDetails(String additionDetails) {
+    this.additionDetails = additionDetails;
+    return this;
+  }
+
+  public String getAdditionDetails() {
+    return additionDetails;
+  }
+  public void setAdditionDetails(String additionDetails) {
+    this.additionDetails = additionDetails;
+  }
+  public Address buildingName(String buildingName) {
+    this.buildingName = buildingName;
+    return this;
+  }
+
+  
+  public String getBuildingName() {
+    return buildingName;
+  }
+  public void setBuildingName(String buildingName) {
+    this.buildingName = buildingName;
+  }
+  public Address street(String street) {
+    this.street = street;
+    return this;
+  }
+
+  public String getStreet() {
+    return street;
+  }
+  public void setStreet(String street) {
+    this.street = street;
+  }
+  
+
+  
+
+  
+  @Override
+  public boolean equals(java.lang.Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (o == null || getClass() != o.getClass()) {
+      return false;
+    }
+    Address address = (Address) o;
+    return Objects.equals(this.tenantId, address.tenantId) &&
+        Objects.equals(this.doorNo, address.doorNo) &&
+        Objects.equals(this.plotNo, address.plotNo) &&
+        Objects.equals(this.id, address.id) &&
+        Objects.equals(this.landmark, address.landmark) &&
+        Objects.equals(this.city, address.city) &&
+        Objects.equals(this.district, address.district) &&
+        Objects.equals(this.region, address.region) &&
+        Objects.equals(this.state, address.state) &&
+        Objects.equals(this.country, address.country) &&
+        Objects.equals(this.pincode, address.pincode) &&
+        Objects.equals(this.additionDetails, address.additionDetails) &&
+        Objects.equals(this.buildingName, address.buildingName) &&
+        Objects.equals(this.street, address.street) ;
+       
+  }
+
+  @Override
+  public int hashCode() {
+    return java.util.Objects.hash(tenantId, doorNo, plotNo, id, landmark, city, district, region, state, country, pincode, additionDetails, buildingName, street);
+  }
+
+  @Override
```

Comment:
lombok annotation is already there to auto-gen all these methods.

Reply:
Changes committed